iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/HalfLineBoundaryImpl.java

```diff
@@ -29,28 +29,28 @@
     public double getV() {
         Terminal t = getConnectable().getTerminal(side);
         Bus b = t.getBusView().getBus();
-        return new SV(t.getP(), t.getQ(), getV(b), getAngle(b)).otherSideU(parent);
+        return new SV(t.getP(), t.getQ(), getV(b), getAngle(b), side).otherSideU(parent);
```

Comment:
I don't understand why you have here `side` and not `Branch.Side.ONE`. Same remark for the three following methods. If I change in the code side by  `Branch.Side.ONE` or  `Branch.Side.TWO`, there is no impact in the unit tests. Same conclusion for dangling lines.

Reply:
@marqueslanauja The `side` here refers to the side of the half line in the tie line. Are you sure that it is the good side and not `SIDE.ONE`?

Reply:
In the halfLine of a TieLine the model end is not always at side ONE. There is a specific field `side` that defines the side associated with the model end. There will not be impact on the unit tests if the shunt admittance at both ends are equal, so that will be the situation of this test.

Reply:
I think should be `side`. After this PR  I will make another one to fix the issues associated with the Tie-Lines (Branch `TieLine-review`). In this new PR I can add some tests to check that  should be `side`.